atom/renderer/atom_sandboxed_renderer_client.cc

```diff
@@ -38,6 +39,11 @@ namespace {
 const std::string kIpcKey = "ipcNative";
 const std::string kModuleCacheKey = "native-module-cache";
 
+bool IsDevTools(content::RenderFrame* render_frame) {
+  return static_cast<GURL>(render_frame->GetWebFrame()->GetDocument().Url())
```

Comment:
Its better to use `blink::WebURL::ProtocolIs` method rather casting it to `GURL`

Reply:
The `WebURL` constructor takes a `GURL`. (https://cs.chromium.org/chromium/src/third_party/WebKit/public/platform/WebURL.h?rcl=daaf59e82687dd66f862f33cb6615b52d18a2d89&l=73)

Reply:
Yes but it is not necessary to do a cast here, when we have a helper method on `WebURL` to verify the scheme.